packages/cli/src/lib/bundler/LinkedPackageResolvePlugin.test.ts

```diff
@@ -50,10 +57,10 @@ describe('LinkedPackageResolvePlugin', () => {
     const callbackX = jest.fn();
     tap(
       {
-        request: '/root/repo/package/x/src/module.ts',
-        path: '/root/repo/package/x/src',
+        request: path.join(root, 'repo', 'package', 'x', 'src', 'module.ts'),
```

Comment:
Another option here is ´path.resolve(root, 'repo/package/x/src/module.ts')´

Reply:
Should we use that? Feels odd to use forward slashes but it's for sure more compact

Reply:
I prefer it :grin:

Resolve essentially does a absolute/relative-path aware join and then normalizes the path, so join is faster but who cares :grin:

Reply:
Sure! I'll just let the current build finish and then if it's green I'll trigger another round with that change.